gazebo_usv.orogen

```diff
@@ -68,3 +69,29 @@ task_context 'TetherSimulationTask' do
 
     port_driven
 end
+
+# Task to limit the propulsion command and output the saturation signal when
+# the inputed command is out of the range
+task_context "ThrusterLimitationTask" do
+    needs_configuration
+
+    # Control limits
+    #
+    # You may only set the speed and effort fields. Setting any other field
+    # will fail on configure
+    #
+    # If set, min and max speeds need to be the opposite of each other. The
+    # controller does not support limiting them separately
```

Comment:
shouldn't you check that?

Reply:
You should at least make it clear that this refers to the live system and that you aim to reproduce its behavior. Even so, I dont think this should come as a limitation in the task itself, meaning that you dont need to throw an exception if these values are not reflections of each other. I would change this comment to a suggestion instead.